qa-tests-backend/src/test/groovy/IntegrationsTest.groovy

```diff
@@ -527,6 +527,31 @@ class IntegrationsTest extends BaseSpecification {
                 Env.mustGetGCPAccessKey()
     }
 
+    @Unroll
+    @Tag("Integration")
+    def "Verify GCS Integration: #integrationName"() {
+        setup:
+        Assume.assumeTrue(!useWorkloadId || Env.HAS_WORKLOAD_IDENTITIES)
+
+        when:
+        "the integration is tested"
+        def backup = ExternalBackupService.getGCSIntegrationConfig(integrationName, useWorkloadId)
+
+        then:
+        "verify test integration"
+        // Test integration for GCS performs test backup (and rollback).
+        assert ExternalBackupService.testExternalBackup(backup)
+
+        where:
+        "configurations are:"
+
+        integrationName                | bucket                     | useWorkloadId
+        // Bucket is in stackrox-ci.
```

Comment:
nit: How about turning this comments to values and use in test name?

Reply:
moved both buckets into `acs-san-stackroxci`.